README.md

```diff
@@ -0,0 +1,3 @@
+`async_scope` -- Creating scopes for non-sequential concurrency.
+
+You can see a rendered copy of the current draft [here](https://htmlpreview.github.io/?https://github.com/kirkshoop/async_scope/blob/gh-pages/asyncscope.html).
```

Comment:
Instead of going to "https://htmlpreview.github.io/?..." we should have been able to access the page directly from https://kirkshoop.github.io/async_scope/asyncscope.html. But somehow that doesn't work. Most probably it's conflicting with https://kirkshoop.github.io.

I believe the link above is ok for our purposes.